reporters/SponsorReporter.js

```diff
@@ -71,11 +80,18 @@ class SponsorReporter {
 
   async generateSponsorsTable() {
     await this.update();
+    console.log(italic(`- There are ${this.empClient.getAllPositions().length} current sponsors`));
     console.log(italic("- All current token sponsors within the specified EMP are printed"));
 
     // For all positions current open in the UMA ecosystem, generate a table.
-    const allPositions = this.empClient.getAllPositions();
     const currentPrice = this.priceFeed.getCurrentPrice();
+    const allPositions = this.empClient.getAllPositions().sort((p1, p2) => {
```

Comment:
This might call for a comment, but it sorts sponsor table by CR threshold:

```
2. Sponsor table💸
- There are 196 current sponsors
- All current token sponsors within the specified EMP are printed
┌────────────────────────────────────────────┬────────────────────────────┬──────────────────────────┬───────────────┐
│                  (index)                   │ Synthetic debt(yUSD-SEP20) │ Backing collateral(WETH) │ Position CR % │
├────────────────────────────────────────────┼────────────────────────────┼──────────────────────────┼───────────────┤
│ 0x7405Fe24003A50e4f4117D35E9b5a9f5E512fEdE │          '100.00'          │         '0.4170'         │   '161.72'    │
│ 0x9D11B1e1445b017902E561ceB201f009b3D53a42 │       '1,302,000.00'       │        '5,533.00'        │   '164.80'    │
│ 0x4Cd52B37fdDD19CcD24B0d0e9a048785C7aaFCEf │        '16,380.00'         │         '81.00'          │   '191.77'    │
│ 0xC0c49B81f9996Ae22BeE60268DFc1e9787e1bB80 │          '500.00'          │          '2.50'          │   '193.91'    │
│ 0xDc04D6CF9C2BE785F5620C353f0254933709ad52 │        '79,190.00'         │         '410.00'         │   '200.79'    │
│ 0x228c1334fc57Eb6E02EcC448E749a041124321c1 │        '225,000.00'        │        '1,199.00'        │   '206.66'    │
│ 0x584C7c9B0A8fB3F648c3912316A2c5E873CF39bb │         '1,100.00'         │          '6.00'          │   '211.53'    │
│ 0x8B64673c9a482913a6e2c1298637532947cD96eE │        '73,000.00'         │         '400.00'         │   '212.50'    │
│ 0xF6E2c94A609Ca303964b65EC991A1A0d2991B961 │        '11,949.00'         │         '65.82'          │   '213.62'    │
│ 0xd165164cbAb65004Da73C596712687C16b981274 │        '98,100.00'         │         '573.93'         │   '226.89'    │
│ 0xdD8B9fCb3a8c619f3Aa688443A095467967cB58D │         '1,133.00'         │          '6.60'          │   '225.91'    │
│ 0x908664ca2D2942665a0e9de85Ba094DA03D816DC │        '15,300.00'         │         '90.00'          │   '228.12'    │
│ 0xe765e0c5f5E6AceA5175a7da83B88c5Dc0f2b437 │         '1,000.00'         │          '6.00'          │   '232.69'    │
│ 0x55a3df57B7AAEC16a162fD5316f35bEC082821CF │        '183,029.00'        │        '1,120.78'        │   '237.48'    │
│ 0x10aab4B0EF76AA2AC9b5909e671517a1171B050E │        '55,000.00'         │         '341.00'         │   '240.44'    │
│ 0xBD1f7d88C76A86C60d41bDDD4819fAe404e7151E │          '201.00'          │          '1.25'          │   '241.18'    │
│ 0x4d1Dcf15AcBC0b69aED7b0d87BDA5Cbc66C48184 │        '21,763.41'         │         '139.09'         │   '247.86'    │
│ 0x8d33d3F0079176d93b34BB98831c7112e4d6C71F │         '3,100.00'         │         '19.80'          │   '247.70'    │
│ 0x2D41CB6623e42E7Bc2DA6cAC3f64374Cd9543F04 │         '1,990.00'         │         '13.00'          │   '253.34'    │
│ 0x689c44E0f4D91AabD7d66B23b29032b665e99b4b │        '12,124.00'         │         '79.00'          │   '252.70'    │
│ 0x8f9bE4310f9AbB0E5843Cc6363908C9B01dfeB3F │         '3,755.00'         │         '24.50'          │   '253.03'    │
│ 0x230d31EEC85F4063a405B0F95bdE509C0d0A8b5D │        '13,300.00'         │         '87.19'          │   '254.24'    │
│ 0xB193Da6F9F2252BE679fd5b020C22DEd01f4eC8D │        '31,500.00'         │         '208.77'         │   '257.04'    │
│ 0x5B93FF82faaF241c15997ea3975419DDDd8362c5 │          '300.00'          │          '2.00'          │   '258.54'    │
│ 0x95A70A517C9E87F0Bd70e380bDE99a41d31D917d │        '697,723.13'        │        '4,672.55'        │   '259.71'    │
│ 0x502FE1105327974B5579785dc7CeD4528a7CE661 │          '859.00'          │          '5.80'          │   '261.85'    │
│ 0x51430536709C685c22600A2672BCEc0cab662F67 │         '3,400.00'         │         '23.00'          │   '262.34'    │
│ 0x7bB0F4314dB3a242B98D789b756e88C0C6d26042 │        '35,560.00'         │         '245.00'         │   '267.19'    │
│ 0xe3cD0FDCc3Efbfd8104B5b44e2D2715031149D07 │        '11,000.00'         │         '76.00'          │   '267.94'    │
│ 0xd53C9Fedcc95187307908D659846a443cb1e7350 │        '18,300.00'         │         '126.00'         │   '267.02'    │
│ 0xF7B10D603907658F690Da534E9b7dbC4dAB3E2D6 │         '6,240.00'         │         '44.00'          │   '273.46'    │
│ 0x41Bc7d0687e6Cea57Fa26da78379DfDC5627C56d │        '140,000.00'        │        '1,000.00'        │   '277.01'    │
│ 0x6595732468A241312bc307F327bA0D64F02b3c20 │        '96,466.00'         │         '700.00'         │   '281.41'    │
│ 0x49739691fB5f3992b3f2536f309D955558e75933 │         '6,900.00'         │         '50.70'          │   '284.96'    │
│ 0xF627E5F4BAd95a956468d8BB6Ee20b119F992E96 │         '1,600.00'         │         '12.00'          │   '290.86'    │
│ 0xff4e2D7169D7D56f8b41C63592FA919b4d1B0165 │          '400.00'          │          '3.00'          │   '290.86'    │
│ 0xD98e44D09660A89022515d6d41662d05A14253e7 │         '4,000.00'         │         '30.00'          │   '290.86'    │
│ 0x62BA33Ccc4a404456e388456C332D871DaE7ae9e │         '3,100.00'         │         '23.35'          │   '292.11'    │
│ 0x5785Fd6a2C4e11a6F3Cce7dAD5700CA9Db63bbdb │        '19,900.00'         │         '150.00'         │   '292.32'    │
│ 0xc442C69F8593C782d0C8bd1F1871CC57175F9D52 │          '400.00'          │          '3.05'          │   '295.71'    │
│ 0x3dD48AD861F0B77D1312F9C95A13e52871Ca95e0 │        '14,108.60'         │         '109.94'         │   '302.23'    │
│ 0x0c8bFC46C4fB39b6eEd7814E8830fAc0a45F8D6D │         '7,200.00'         │         '57.00'          │   '307.02'    │
│ 0xffd4DAe0d7D8ddb6F408dca0a47763Ae3A57F4Ce │        '12,554.00'         │         '100.00'         │   '308.92'    │
│ 0x2B53f9CB996a3C0b76B4E53104Ed975aAD720c70 │        '61,510.00'         │         '488.57'         │   '308.04'    │
│ 0x25Ae7b45D8646580dfcAE403D29164729eB8642f │        '196,780.00'        │        '1,567.30'        │   '308.88'    │
│ 0x180780701cfED2151cf6eD76580949D902b41790 │          '880.00'          │          '7.00'          │   '308.49'    │
│ 0xfe32A37F15ee4a4b59715530E5817D1322B9df80 │          '100.00'          │         '0.8000'         │   '310.25'    │
│ 0x0CAC772568B899104677B4fa2B27ec4056548f02 │          '250.00'          │          '2.00'          │   '310.25'    │
│ 0x5C7c6d069ba232718f37C27A9549b547C359E31C │        '16,250.00'         │         '130.00'         │   '310.25'    │
│ 0xA423fE4CFb811E9CF6a61a02e80E372c0970d4b0 │        '38,700.00'         │         '310.00'         │   '310.65'    │
│ 0xc7777C1a0Cf7E22c51b44f7CeD65CF2A6b06dc5C │         '2,090.00'         │         '16.69'          │   '309.74'    │
│ 0x526C7665C5dd9cD7102C6d42D407a0d9DC1e431d │        '23,787.00'         │         '190.00'         │   '309.77'    │
│ 0xAc7e04b9B3757cd94EAEeC4Abaa5b754EC2C0f88 │         '1,700.00'         │         '13.57'          │   '309.75'    │
│ 0x71F12a5b0E60d2Ff8A87FD34E7dcff3c10c914b0 │        '69,000.00'         │         '550.00'         │   '309.13'    │
│ 0xbA766A08f0a126Ed43B2699F722d2063e5fBb308 │         '3,500.00'         │         '28.00'          │   '310.25'    │
│ 0xc1aD79aB00b3F63577ca653CdFDa344e9847D4d0 │        '51,000.00'         │         '408.00'         │   '310.25'    │
│ 0x9224bbB4E0fBE2f2f8fab55DEbC41Eb21fDfB804 │          '250.00'          │          '2.00'          │   '310.25'    │
│ 0x605ACc13c07CB2dE5261dc64D315857fDE7d5C5c │          '470.00'          │          '3.77'          │   '311.08'    │
│ 0x829c66804F6D94360EfF112CD1ECaE0972d736ea │        '12,500.00'         │         '100.00'         │   '310.25'    │
│ 0x57db5D6aa783Cf29af41330569D24957140FD3eB │          '390.00'          │          '3.14'          │   '312.48'    │
│ 0xF6853c77a2452576EaE5af424975a101FfC47308 │         '8,374.00'         │         '67.42'          │   '312.23'    │
│ 0xBe3BF9Bb42bF423695B792250BE206C9A58e2dE1 │         '9,250.00'         │         '75.00'          │   '314.44'    │
│ 0x148171f2598078f807DdE899De39B639F4a811B5 │         '1,701.00'         │         '13.80'          │   '314.63'    │
│ 0x505846a0a89dd26fA5cD0677FD5406039c218620 │         '4,320.39'         │         '35.24'          │   '316.37'    │
│ 0x4B1e0388233B41B1225380e291cCa33F38c7A49b │         '4,285.00'         │         '35.05'          │   '317.28'    │
│ 0x55C44b2827D73BA7eDaFf4FE21f7E12F57e4115b │          '100.00'          │         '0.8200'         │   '318.01'    │
│ 0xA631Df8625Ad8c287834c79D1Fa6F32086Cb23c3 │         '2,200.00'         │         '18.00'          │   '317.30'    │
│ 0x130daD1E7981AbF598aaEc5ABef9f38b1d9EBD98 │         '3,550.00'         │         '29.00'          │   '316.81'    │
│ 0x2D0C7f2E4475aD0a3df13c47F2992e995Bf9364F │          '568.48'          │          '4.65'          │   '317.88'    │
│ 0xBDf37957660F4D444361526C4FBc9aAedFa32632 │         '1,625.00'         │         '13.30'          │   '317.41'    │
│ 0x69F329EE79C98A94765e4105aDFd5a87458d919C │         '8,100.00'         │         '66.64'          │   '319.10'    │
│ 0x78e96Be52e38b3FC3445A2ED34a6e586fFAb9631 │        '58,800.00'         │         '486.67'         │   '320.98'    │
│ 0x366158b891147400FAF0010C6eA4151AD3bE0499 │         '3,288.00'         │         '27.21'          │   '320.96'    │
│ 0x7db7da086318462a46327b8F2d46f457008CC111 │         '6,180.00'         │         '51.00'          │   '320.04'    │
│ 0x84740F97Aea62C5dC36756DFD9F749412534220E │        '664,890.00'        │        '5,521.00'        │   '322.03'    │
│ 0x1B72Bac3772050FDCaF468CcE7e20deb3cB02d89 │        '30,070.00'         │         '250.00'         │   '322.43'    │
│ 0x5595F9E14a9A3B957d08E904939AA351e265C157 │         '3,604.00'         │         '29.88'          │   '321.53'    │
│ 0xedc29eF8B58CD8ed25cAA2B92AAB8A5547312718 │          '239.00'          │          '2.00'          │   '324.53'    │
│ 0x30D8058D120eB971646a14aF5C0A0b678ba533dA │          '359.50'          │          '3.00'          │   '323.63'    │
│ 0x25125E438b7Ae0f9AE8511D83aBB0F4574217C7a │        '29,300.00'         │         '244.90'         │   '324.15'    │
│ 0x325C8953add0FBa9deA7F65fAD6eA44774274e07 │        '180,000.00'        │        '1,500.00'        │   '323.18'    │
│ 0x3c5d1A46c8988fA19F28aD5531c039f51D7E8C09 │         '4,300.00'         │         '35.80'          │   '322.88'    │
│ 0xc5aA216ab58291b7cFe9be87c0e46917DE032907 │         '1,200.00'         │         '10.00'          │   '323.18'    │
│ 0xbE09d4cD0025A5217626852b21576590e3A62d23 │         '2,400.00'         │         '20.00'          │   '323.18'    │
│ 0x9C2579454B9854FF3dFB64e9Acf687615Ee99856 │         '3,848.00'         │         '32.00'          │   '322.51'    │
│ 0x1F92fC15bCbEB4dD24EeFbead8d7aeE409f891DC │         '1,200.00'         │         '10.00'          │   '323.18'    │
│ 0x323AFc6d2DbE2Eb0AE71Ed1f87db8956834E861d │         '1,200.00'         │         '10.00'          │   '323.18'    │
│ 0xEF22364cdabD7F39a3B6Fd836f059336c810A8a9 │          '120.00'          │          '1.00'          │   '323.18'    │
│ 0xF3Ae3bBdeB2fB7F9C32FbB1F4fbDAF1150a1c5Ce │        '60,000.00'         │         '500.00'         │   '323.18'    │
│ 0x3c229864378b3B0adF98aE247Ca33986AED8EC1E │         '2,400.00'         │         '20.00'          │   '323.18'    │
│ 0x03d15cd9EedEcbEC37BAE097C98d53438Ad83E84 │        '12,000.00'         │         '100.00'         │   '323.18'    │
│ 0xC37c07Aa97248726884018eBbdC006601C34B0BA │         '1,100.00'         │          '9.20'          │   '324.35'    │
│ 0xc1228273C77a4939bCc42A0D8f7B0077228e7F3c │          '238.00'          │          '1.99'          │   '325.86'    │
│ 0xF0F32C3c8DEf7b43caA14fEd5752AbC5381C8374 │         '3,560.00'         │         '30.00'          │   '326.81'    │
│ 0x5eC43C257416C7D7dc9216018E7B430a8dd2eDdC │          '950.00'          │          '8.00'          │   '326.58'    │
│ 0xE030fe0291c0DbFCaD873860d816312422017565 │        '115,005.00'        │         '966.00'         │   '325.75'    │
│ 0xFBb42577EB5E6c40AF214Bc457afE1922f55556b │          '811.00'          │          '6.85'          │   '327.56'    │
│ 0x4d712089718ead30C57C892E29925b34BaA10CDd │         '9,660.00'         │         '81.90'          │   '328.80'    │
│ 0xAb3D86FCB9f602c3762635204B6062dF4D2450d6 │         '6,000.00'         │         '51.00'          │   '329.64'    │
│ 0x1178463F809657513Bfaa2bb69DBEF55d9f61e9d │          '118.00'          │          '1.00'          │   '328.66'    │
│ 0x731Cf08ca9442b1E59080c41002F15Ceb32fD8eb │        '23,300.00'         │         '198.71'         │   '330.75'    │
│ 0x152676C60B6e1c4e70E197444217ab9dC0737A1b │          '702.00'          │          '6.00'          │   '331.47'    │
│ 0xd0dC07B98769f23A7BDbef15A35Faa256CB65dCF │          '350.00'          │          '3.00'          │   '332.59'    │
│ 0x9ae57ce31BDae73A0d3BF701998Af5494a1cEeec │        '41,550.00'         │         '360.00'         │   '336.01'    │
│ 0x56D14fcb4587B5845fA4823aF790672EBF92580d │         '1,500.00'         │         '13.00'          │   '336.11'    │
│ 0x8CF3fa9773E05B10C7472F53342043376DFC518b │         '1,269.00'         │         '11.00'          │   '336.17'    │
│ 0x5d47e5D242a8F66a6286b0a2353868875F5d6068 │          '678.92'          │          '5.90'          │   '337.23'    │
│ 0x85AcC988d311Db62cf45280C3174758c9681B8E2 │          '575.00'          │          '5.00'          │   '337.23'    │
│ 0xCdDAD4fDaa65e7ab74DDA9e024f887A6765CAFbb │          '100.00'          │         '0.8700'         │   '337.40'    │
│ 0x3a39F8d526561768B72fde1a17E928d9B1FC0c64 │         '2,865.00'         │         '25.00'          │   '338.41'    │
│ 0x3Caba50E426de2003f937Ac961BD6834B206fb1C │         '2,275.00'         │         '20.00'          │   '340.94'    │
│ 0x6A8Ff90E596E037b2Ded3881c19Ab4Fd1727A0c8 │          '100.37'          │         '0.8804'         │   '340.19'    │
│ 0xDd78F6a187f07Fa4672Fdd2aac32E6A2Ffd62023 │          '800.00'          │          '7.00'          │   '339.34'    │
│ 0xe16a5b9026779Bf8DE3Ac8C70dDA9de76F5dd9Cc │          '625.00'          │          '5.50'          │   '341.28'    │
│ 0x08bb933f6560D28C23e14523B93c8544ad6A92fd │        '12,860.00'         │         '113.00'         │   '340.77'    │
│ 0xEC005B6eDbA4eb0a230317921555BC8D548D8224 │        '39,751.00'         │         '349.00'         │   '340.49'    │
│ 0xC31dB2E710192791b65dE43D4B84886A6d770322 │        '58,200.00'         │         '514.00'         │   '342.50'    │
│ 0x7088C5611dAE159A640d940cde0a3221a4af8896 │         '2,720.00'         │         '24.00'          │   '342.19'    │
│ 0x03525780478bf6181B2FDD82df9aB6E7f26a8Ffe │         '1,770.00'         │         '15.60'          │   '341.84'    │
│ 0x2B83aC024B0a4Ba3f93776Cacb61D98310614aFE │          '355.00'          │          '3.13'          │   '341.93'    │
│ 0xb8F2A55d24bf040F66969cBCEbe2EA2C4927515D │       '1,218,120.00'       │       '10,772.00'        │   '342.95'    │
│ 0xA5a69107816c5E3dfA5561E6B621dFE6294F6e5B │          '170.00'          │          '1.50'          │   '342.19'    │
│ 0xC5fdA9b6258b392f120dB67202C605BF22c29853 │         '4,500.00'         │         '40.00'          │   '344.72'    │
│ 0xf94571dbdFf33446DAbD17040Cd6236B0D2C2545 │         '2,422.00'         │         '21.50'          │   '344.26'    │
│ 0xbC90B3Ce40fc3Ed921D910f3e046C65954fFF7cB │        '45,000.00'         │         '400.00'         │   '344.72'    │
│ 0x15BDd87fE60E212C150d18D205766bDA86350152 │        '10,000.00'         │         '89.00'          │   '345.15'    │
│ 0x38f48a25D6c156beA5e1F7F6287BCba4206f94A2 │          '900.00'          │          '8.00'          │   '344.72'    │
│ 0xd788d826864dfFBc61C75F0c85EeEb840f246247 │        '14,000.00'         │         '125.00'         │   '346.26'    │
│ 0x37Fa7c6B28fc5545d9523D6dbb329Be7e1B9e7A0 │         '2,350.00'         │         '21.00'          │   '346.56'    │
│ 0x9b0c19000a8631c1f555bb365bDE308384E4f2Ff │         '2,900.00'         │         '26.00'          │   '347.70'    │
│ 0xdE8d36B1A157dbd62Ab9983F82dd85B1eCC92025 │         '6,335.00'         │         '56.77'          │   '347.58'    │
│ 0xCBeC4e7483F3e2E2A814Fda4F5cC6C77f6e30417 │        '45,000.00'         │         '403.00'         │   '347.31'    │
│ 0x9217b8D0e7E831cbf2366476Ad1C09204F4772f0 │         '5,555.00'         │         '50.00'          │   '349.07'    │
│ 0x9555C99451dc1c1cdcd0cb224D15F6972710d0d7 │          '500.00'          │          '4.50'          │   '349.03'    │
│ 0x22d7D38717474569Ea63cBDa595F4BbC0f1B640D │          '100.00'          │         '0.9000'         │   '349.03'    │
│ 0x165Dc87a77B08b289961689CeC85bdB1c9E34AfD │          '100.00'          │         '0.9000'         │   '349.03'    │
│ 0x9eD629Bc305c5731D43de27Cbbe8F77976893c2b │        '74,000.00'         │         '670.00'         │   '351.13'    │
│ 0x840b924613F78b4d5f243552cA0C9557288D7614 │          '120.00'          │          '1.08'          │   '351.33'    │
│ 0xDaf3f7f611eFA83033155D23C3EbC5B52dDBAeBb │         '3,300.00'         │         '30.00'          │   '352.56'    │
│ 0x334CE923420ff1aA4f272e92BF68013D092aE7B4 │         '1,100.00'         │         '10.00'          │   '352.56'    │
│ 0xf0a8CEA0DE817eec61b4d27456CFB003167E8024 │         '1,100.00'         │         '10.00'          │   '352.56'    │
│ 0x744b130afb4E0DFb99868B7A64a1F934b69004C4 │          '825.00'          │          '7.50'          │   '352.56'    │
│ 0x95861c41E7767Dc737bca90b175AD51e5F7B9ADA │          '115.00'          │          '1.04'          │   '352.47'    │
│ 0x92cfdcEEb276a6Bc2f13B6907CD8E6aC245dD0FE │         '2,850.00'         │         '26.00'          │   '353.80'    │
│ 0x525A502eEA5599B03372851174D9A8dCEBff68b6 │        '10,000.00'         │         '91.00'          │   '352.91'    │
│ 0x203a67b09429d1BAA2c56Eb4C57634E498F07819 │        '34,042.00'         │         '311.55'         │   '354.93'    │
│ 0x5890eFb07d3900FF15f353B72Fddc48759356290 │         '2,400.00'         │         '22.00'          │   '355.50'    │
│ 0xA24Ebe62c3E7167c87C6915081815A6B82d45a44 │         '1,090.00'         │         '10.00'          │   '355.79'    │
│ 0xa2e44E28ABc49b83AF06151C630a3B89C65DF4f6 │          '100.00'          │         '0.9315'         │   '361.25'    │
│ 0x19eE7696D2a71548A7f1FC9cef644460d7e73297 │         '1,500.00'         │         '14.00'          │   '361.96'    │
│ 0x53Ec88165201F2c59A8ADDF169a0b9eb6a3d18af │          '225.00'          │          '2.10'          │   '361.96'    │
│ 0x538DeD4D0d461206AE8E0c021d5179D3a31D2b12 │         '7,792.00'         │         '73.00'          │   '363.33'    │
│ 0x1FC432eB5e873A12E909Da65ad9E14be05F1139B │        '50,000.00'         │         '470.00'         │   '364.55'    │
│ 0x4F9a04Bf67a65A59Ef0beB8dcC83f7f3cC5C5D23 │         '3,500.00'         │         '33.00'          │   '365.65'    │
│ 0xcD1c84F035788AB653CE5351461f53e6AF86A419 │        '25,350.00'         │         '240.00'         │   '367.16'    │
│ 0xC76c067f1864C610880cf730eb69D91f223f1E79 │        '10,000.00'         │         '95.00'          │   '368.42'    │
│ 0x8159c2a9C7fB01CB57441125b33Da62A69EAB42e │          '100.00'          │         '0.9500'         │   '368.42'    │
│ 0xe38EF7739775EC8c9Ad62b759eb6aa9F047d84eD │          '105.00'          │          '1.00'          │   '369.35'    │
│ 0x4eBBbFF7E4FB62CD751c9402d10a67Fd763eF825 │         '2,300.00'         │         '22.00'          │   '370.95'    │
│ 0xfF3fc772434505ABff38EEcDe3C689D4b0254528 │         '2,000.00'         │         '19.24'          │   '373.26'    │
│ 0xfeF3DBD11B9989a6269850B199d31076899C41Ba │         '3,000.00'         │         '29.00'          │   '374.89'    │
│ 0x81f7EF7586c7Da8fDB0BB94032F3ad06ae21f644 │          '100.00'          │          '1.00'          │   '387.82'    │
│ 0xD9A4693bc2D2b0246F931CD307d2bc53328cd73D │        '30,000.00'         │         '300.00'         │   '387.82'    │
│ 0x8d21e3C8993633546E89cb4A24e5c0A5BC4D97E5 │        '25,000.00'         │         '250.00'         │   '387.82'    │
│ 0xc034AbCD15869f440303F657392BF8Db93CF8821 │         '1,500.00'         │         '15.00'          │   '387.82'    │
│ 0x099686478448CE02F8E61C6b01904d0548daeAe5 │          '250.00'          │          '2.50'          │   '387.82'    │
│ 0xE233CEc774061e9871b865c59d1df66Be52E3388 │         '2,000.00'         │         '20.00'          │   '387.82'    │
│ 0x88D5f7b72e35b869b0051652c1BEF4c1297f6b5c │          '100.00'          │          '1.00'          │   '387.82'    │
│ 0x2bA117929C3c53ac7C38AD870321573c740Cf139 │         '2,000.00'         │         '20.00'          │   '387.82'    │
│ 0xfDc6a17917115498A3Ed417D4A89224685f26EA3 │        '10,000.00'         │         '100.00'         │   '387.82'    │
│ 0x836582A4d3Fa08066F71D03D79413759d40A551C │          '100.00'          │          '1.00'          │   '387.82'    │
│ 0xf17e6b621861990f096b60172b3d700FA9CCeC85 │         '2,000.00'         │         '20.00'          │   '387.82'    │
│ 0x239EeC9EC218f71cEf5CC14D88b142ed4fF44110 │        '10,000.00'         │         '100.00'         │   '387.82'    │
│ 0x11b67a503B3B702104EeeF69a4eA1365D0F2c658 │         '4,000.00'         │         '40.00'          │   '387.82'    │
│ 0xa311f7Ca3Eb2Fb98123A807f9b8e4BbbDBDcB2Ee │         '5,000.00'         │         '50.00'          │   '387.82'    │
│ 0x2C99d50Eb4DAA1D26b5bA0b1dAaB58cBFc0AC3E6 │        '10,000.00'         │         '100.00'         │   '387.82'    │
│ 0xdaC071eDBe067499281F1272cf39F4CCB1e50ac8 │        '40,000.00'         │         '400.00'         │   '387.82'    │
│ 0x5444a43aea36f3CF015AaF24a938Ae6bAEE2614D │         '2,000.00'         │         '20.00'          │   '387.82'    │
│ 0xE6662ec7FC7c84231B29F0f0da6321673d100854 │         '3,000.00'         │         '30.00'          │   '387.82'    │
│ 0xc6fE438c3d3e9b9E18e6fa47921B04ECa19dCC57 │          '140.00'          │          '1.40'          │   '387.82'    │
│ 0xD2E8827d4b1c44f64d1FA01BfBC14DC8545eCa41 │          '100.00'          │          '1.00'          │   '387.82'    │
│ 0x2983A89E018bae9E3EEC745Cced1bE85a59C44c4 │          '100.00'          │          '1.00'          │   '387.82'    │
│ 0x73eB7A4756eF34B903d8f0D320138FA4f2e4Ec5b │          '500.00'          │          '5.00'          │   '387.82'    │
│ 0x35C48803A0Ce972Db14C21Db7310379A14E225E9 │          '100.00'          │          '1.00'          │   '387.82'    │
│ 0xAABd5fBcb8ad62D4FbBB02a2E9769a9F2EE7e883 │          '200.00'          │          '2.00'          │   '387.82'    │
│ 0x8a9D46d28003673Cd4FE7a56EcFCFA2BE6372e64 │         '1,500.00'         │         '15.00'          │   '387.82'    │
│ 0x29c7B44E0584624C1e877d3eE0856520e2851Ba6 │        '20,000.00'         │         '204.00'         │   '395.57'    │
│ 0x7e1FEb23C08bFe287f6d3350281029aF0889502a │          '300.00'          │          '3.10'          │   '400.74'    │
│ 0x88aAF4a728e14748cea5e9A3cA1306Ac567ba472 │        '20,000.00'         │         '211.41'         │   '409.94'    │
│ 0x57Ef012861c4937A76B5D6061bE800199A2b9100 │        '28,000.00'         │         '300.60'         │   '416.36'    │
│ 0x821305E72A0E9545Ae3E29daEE68eDac509d442F │          '759.04'          │          '8.70'          │   '444.51'    │
│ 0x2F34ee226e149a1Fe2d03e883DF9544d84c1c47C │         '5,300.00'         │         '63.97'          │   '468.14'    │
│ 0x8a83716acd66D9e1fb18c9b79540B72E04f80aC0 │         '4,400.00'         │         '55.00'          │   '484.77'    │
│ 0xa9Eb5209Ae0C5184d9e8126F658ec576206BE00A │         '4,000.00'         │         '50.00'          │   '484.77'    │
│ 0x1dF428833f2C9FB1eF098754e5D710432450d706 │          '150.00'          │          '2.00'          │   '517.09'    │
│ 0x775C10C93e0db7205B2643458233c64fC33Fd75b │          '100.00'          │          '2.00'          │   '775.64'    │
└────────────────────────────────────────────┴────────────────────────────┴──────────────────────────┴───────────────┘
```

Reply:
looks sweet

Reply:
+1, this is a great idea.